backend/app/main.py

```diff
@@ -171,6 +171,18 @@ async def shatter_parent(
     return result
 
 
+@app.patch(
```

Comment:
H: We were imagining implementing this by dropping the partition on assignments rather than deleting the individual records. Should be more performant and clean up not having a bunch of empty assignment partitions lying around.

Reply:
+1 this is the preferred method; note that we'll want to recreate the partition also-- we can't insert records into a partition that doesn't exist. 

So I'd do a DROP TABLE <partition_name> and then another CREATE TABLE ... PARTITION OF

Reply:
Makes sense -- thanks!

Reply:
This has been updated to implement this approach.